### PR TITLE
fix: signature of Sink implementation

### PR DESCRIPTION
**Pull request by tpluscode**
brings the class up to date with changes from DefinitelyTyped/DefinitelyTyped#41591

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -85,7 +86,7 @@
     "version": "manual-git-changelog onversion"
   },
   "dependencies": {
-    "@rdfjs/data-model": "^1.1.1",
+    "@rdfjs/data-model": "^1.1.2",
```
Comment (tpluscode) [inline package.json]: Not sure if that's desired but probably doesn't hurt 🤷‍♂

File: lib/RdfXmlParser.ts [hunk 2]
```diff
@@ -172,9 +172,9 @@ export class RdfXmlParser extends Transform {
   /**
    * Parses the given text stream into a quad stream.
    * @param {NodeJS.EventEmitter} stream A text stream.
-   * @return {NodeJS.EventEmitter} A quad stream.
+   * @return {RDF.Stream} A quad stream.
    */
-  public import(stream: EventEmitter): EventEmitter {
+  public import(stream: EventEmitter): RDF.Stream {
```
Comment (rubensworks) [inline lib/RdfXmlParser.ts]: Shouldn't this be the other way around? Input RDF.Stream and output EventEmitter?

Comment (tpluscode) [reply lib/RdfXmlParser.ts]: Nope. It's a parser so you parse a basic stream and produce a quad stream.

Comment (rubensworks) [reply lib/RdfXmlParser.ts]: Yes, of course...

**Comment by rubensworks** [comment]
Thanks! Released as 1.3.3.

**Pull request merged**